exchange/exchangetest/passthrough.json

```diff
@@ -0,0 +1,122 @@
+{
+    "passthrough_ext_flag": true,
+    "incomingRequest": {
+        "ortbRequest": {
+            "id": "some-request-id",
+            "site": {
+                "page": "test.somepage.com"
+            },
+            "imp": [{
+                "id": "my-imp-id",
+                "video": {
+                    "mimes": ["video/mp4"]
+                },
+                "ext": {
+                    "appnexus": {
+                        "placementId": 1
+                    },
+                    "prebid": {
+                        "passthrough": {
+                            "passthrough_val": 20
+                        }
+                    }
+                }
+            }],
+            "ext": {
+                "prebid": {
+                    "passthrough": {"passthrough_val":20}
```

Comment:
Perhaps the two occurrences of `passthrough_val` in the incoming request (on lines 20 and 27) should have different values so we can be sure they are being copied to the right places in the outgoing request and response?